### DISCUSSION TITLE
github pages/Settings

### DISCUSSION THREAD

**Discussion by Roberto-acunha** [community/community#36696] [New to GitHub]
Me ajude  a entender porque o github pages não aparece dentro de setting  ?
obrigado !

**Comment by ettaboyle** [comment]
Thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by Bashamega** [comment]
It will appear in your repository settings. Or if you can't find it you can got to this URL, but be sure that you change your user name repositry name  you can got to https://github.com/Your-username/Repositry-name/settings/pages

**Comment by DSDmark** [comment]
### Hey @Roberto-acunha,
 If you want to **delete, unpublish or republish,** then you can do that inside the **settings of your repo** like this:-

- ![Screenshot from 2022-10-20 21-10-09](https://user-images.githubusercontent.com/85744816/197001894-ebb0a1cf-324f-44b4-ac39-2aec23826412.png)

---

### then :-
- ![Screenshot from 2022-10-20 21-09-51](https://user-images.githubusercontent.com/85744816/197001549-c2dedc11-4969-40cd-88b3-7f08360ae05d.png)
